geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/main/scala/org/locationtech/geomesa/parquet/io/SimpleFeatureReadSupport.scala

```diff
@@ -181,12 +216,12 @@ object SimpleFeatureReadSupport {
     override def addBinary(value: Binary): Unit = callback.set(index, value.getBytes)
   }
 
-  class ListConverter(index: Int, binding: Seq[ObjectType], callback: Settable) extends GroupConverter {
+  class ListConverter(binding: ObjectType, index: Int, callback: Settable) extends GroupConverter {
```

Comment:
Any particular reason for the argument order change?

Reply:
i changed it from a Seq to a single type (same with the map converter below), to make it a little clearer what's required. Once the signature had changed, seemed like the binding was 'higher importance' so i put it first... no real reason.